service/controller/clusterapi/v19/key/common.go

```diff
@@ -19,6 +21,14 @@ func IsDeleted(getter DeletionTimestampGetter) bool {
 	return getter.GetDeletionTimestamp() != nil
 }
 
+func KubeConfigClusterName(getter LabelsGetter) string {
+	return fmt.Sprintf("giantswarm-%s", ClusterID(getter))
+}
+
+func KubeConfigSecretName(getter LabelsGetter) string {
+	return fmt.Sprintf("kubeconfig-%s", ClusterID(getter))
```

Comment:
I changed this. It was `%s-kubeconfig`. Is that a problem? I didn't figure out how this could break. I looked in `app-operator` but I couldn't find anything related to that there.

Reply:
When `app` CR is created it has `kubeconfig` default to `%s-kubeconfig` secret. We should change this name part in `api` before you roll out `clusterapi` controller into every installation that are using `app` CR currently. I think it's not easy to sync bring all these together, and yet you too have to look into the secret resource left behind. 
Can we go with %s-kubeconfig at the moment?

Reply:
Totally, I will just change this back and not bother. Sounds way to complicated to do this.